<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DineshManoharan" failures="43" tests="113" name="Default test" time="1192.026" errors="0" timestamp="2024-04-22T19:06:02 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.101"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.896"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="16.275">
    <failure type="java.lang.AssertionError" message="expected [Enter all details] but found [please upload your Profile picture]">
      <![CDATA[java.lang.AssertionError: expected [Enter all details] but found [please upload your Profile picture]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.AdminModule.check_how_it_behave_in_admin_page_whether_it_shows_error_message_or_not(AdminModule.java:111)
at ✽.Check how it behave in admin page whether it shows error message or not.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College10_Admin.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.446">
    <failure type="java.lang.Exception" message="Values Overriding">
      <![CDATA[java.lang.Exception: Values Overriding
at stepDefinitions.AdminModule.refresh_the_page_and_check(AdminModule.java:116)
at ✽.Refresh the page and check.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College10_Admin.feature:23)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="13.883"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="14.088"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="14.182"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="13.142"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="12.426"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="12.339"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.208">
    <failure type="java.lang.Exception" message="Passqord values didn&amp;apos;t shows">
      <![CDATA[java.lang.Exception: Passqord values didn't shows
at stepDefinitions.AdminModule.check_all_values_are_present_in_admin(AdminModule.java:198)
at ✽.Check all values are present in admin(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College10_Admin.feature:54)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="13.059">
    <failure type="java.lang.Exception" message="Permission Validation Must">
      <![CDATA[java.lang.Exception: Permission Validation Must
at stepDefinitions.AdminModule.if_it_allow_it_s_bug(AdminModule.java:217)
at ✽.If it allow its bug.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College10_Admin.feature:60)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.107"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.939"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.604">
    <failure type="java.lang.AssertionError" message="expected [Enter the credentials] but found [please upload the student data]">
      <![CDATA[java.lang.AssertionError: expected [Enter the credentials] but found [please upload the student data]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.SupportModule.check_how_it_behave_in_support_page_whether_it_shows_error_message_or_not(SupportModule.java:95)
at ✽.Check how it behave in support page whether it shows error message or not.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College11_Support.feature:20)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="13.742">
    <failure type="java.lang.Exception" message="Error message didn&amp;apos;t throw in support page">
      <![CDATA[java.lang.Exception: Error message didn't throw in support page
at stepDefinitions.SupportModule.check_how_its_behave_in_support_page(SupportModule.java:112)
at ✽.Check how its behave in support page.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College11_Support.feature:25)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="16.039">
    <failure type="java.lang.Exception" message="Set limit video mb">
      <![CDATA[java.lang.Exception: Set limit video mb
at stepDefinitions.SupportModule.check_whether_the_videos_has_been_upload_oe_not(SupportModule.java:131)
at ✽.Check whether the videos has been upload oe not.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College11_Support.feature:30)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.071">
    <failure type="java.lang.Exception" message="Searcch Box didn&amp;apos;t Work">
      <![CDATA[java.lang.Exception: Searcch Box didn't Work
at stepDefinitions.SupportModule.check_all_words_contains_v_char_in_support(SupportModule.java:140)
at ✽.Check All words contains V char in support.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College11_Support.feature:34)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="15.303"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.714"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.203">
    <failure type="java.lang.AssertionError" message="expected [Enter the credentials] but found [Exam ID is already exist]">
      <![CDATA[java.lang.AssertionError: expected [Enter the credentials] but found [Exam ID is already exist]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.ExamModule.check_how_it_behave_in_exam_page_whether_it_shows_error_message_or_not(ExamModule.java:125)
at ✽.Check how it behave in exam page whether it shows error message or not.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College12_Exam.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="15.588"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="21.673"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="13.572">
    <failure type="java.lang.Exception" message="Existing Time and date doesn&amp;apos;t want to shows">
      <![CDATA[java.lang.Exception: Existing Time and date doesn't want to shows
at stepDefinitions.ExamModule.throw_errors(ExamModule.java:221)
at ✽.Throw Errors.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College12_Exam.feature:34)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="13.411">
    <failure type="java.lang.Exception" message="Submit and add button didn&amp;apos;t work properly">
      <![CDATA[java.lang.Exception: Submit and add button didn't work properly
at stepDefinitions.ExamModule.click_submit_and_add_icon_and_check_how_it_behave(ExamModule.java:251)
at ✽.Click submit and add icon and check how it behave.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College12_Exam.feature:39)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="16.379"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.336">
    <failure type="java.lang.Exception" message="Without date how can we schedule the exam?">
      <![CDATA[java.lang.Exception: Without date how can we schedule the exam?
at stepDefinitions.ExamModule.check_its_created_or_not_in_exam_page(ExamModule.java:282)
at ✽.Check its created or not in exam page.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College12_Exam.feature:49)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.534"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.015">
    <failure type="java.lang.Exception" message="Dont allow negative values in mark fiels">
      <![CDATA[java.lang.Exception: Dont allow negative values in mark fiels
at stepDefinitions.ExamModule.check_the_error_message_shows_or_not_related_to_mark_in_exam_module(ExamModule.java:314)
at ✽.Check the error message shows or not related to mark in exam module.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College12_Exam.feature:59)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.122">
    <failure type="java.lang.Exception" message="Enter irrelevant answer how it provide marks">
      <![CDATA[java.lang.Exception: Enter irrelevant answer how it provide marks
at stepDefinitions.ExamModule.if_its_allow_its_bug(ExamModule.java:342)
at ✽.If its allow its bug.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College12_Exam.feature:64)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.4"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.806"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.022"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.603"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="17.095"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="17.173">
    <failure type="java.lang.AssertionError" message="expected [22/04/2024] but found [29/03/2024]">
      <![CDATA[java.lang.AssertionError: expected [22/04/2024] but found [29/03/2024]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.BranchModule.check_the_created_date_and_time(BranchModule.java:123)
at ✽.Check the created date and time.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College1_Branch.feature:22)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.23"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="16.374">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="stale element reference: stale element not found in the current frame
  (Session info: chrome=124.0.6367.61)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Build info: version: &amp;apos;4.11.0&amp;apos;, revision: &amp;apos;040bc5406b&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;19.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [1afb920afdf99db8296b28cf409ba88b, clickElement {id=f.6D0BEC7700AE33B001CFB83F50D30AF3.d.9BBFF814243E5AA19E077659E01937DE.e.82}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 124.0.6367.61, chrome: {chromedriverVersion: 124.0.6367.60 (8771130bd84f..., userDataDir: C:\Users\dinesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50614}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50614/devtoo..., se:cdpVersion: 124.0.6367.61, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (1afb920afdf99db8296b28cf409ba88b)] -&amp;gt; xpath: /html/body/div/div/div[1]/div[2]/div[2]/div/form/div[1]/label/span/div]
Session ID: 1afb920afdf99db8296b28cf409ba88b">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: stale element reference: stale element not found in the current frame
  (Session info: chrome=124.0.6367.61)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Build info: version: '4.11.0', revision: '040bc5406b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '19.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [1afb920afdf99db8296b28cf409ba88b, clickElement {id=f.6D0BEC7700AE33B001CFB83F50D30AF3.d.9BBFF814243E5AA19E077659E01937DE.e.82}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 124.0.6367.61, chrome: {chromedriverVersion: 124.0.6367.60 (8771130bd84f..., userDataDir: C:\Users\dinesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50614}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50614/devtoo..., se:cdpVersion: 124.0.6367.61, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (1afb920afdf99db8296b28cf409ba88b)] -> xpath: /html/body/div/div/div[1]/div[2]/div[2]/div/form/div[1]/label/span/div]
Session ID: 1afb920afdf99db8296b28cf409ba88b
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:224)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:77)
at Modules.BranchFields.uploadImage(BranchFields.java:49)
at stepDefinitions.BranchModule.enter_the_all_credentials_like1(BranchModule.java:144)
at ✽.Enter the all credentials like "dsds" "faa1411" "fa@gamil.com" "243131" "4352" "12345" "dfbdbd" "fdsfbdfs" "iokhjkj" "hghvbjj" "789" "0998"(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College1_Branch.feature:33)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="16.317">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="stale element reference: stale element not found in the current frame
  (Session info: chrome=124.0.6367.61)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Build info: version: &amp;apos;4.11.0&amp;apos;, revision: &amp;apos;040bc5406b&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;19.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [f556026a5996b2363f8140bb0913e400, clickElement {id=f.C0E64AD947F7FF20866C745F523CFD56.d.CD7FBF3D028BB7C7E9997D5F610F4B17.e.89}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 124.0.6367.61, chrome: {chromedriverVersion: 124.0.6367.60 (8771130bd84f..., userDataDir: C:\Users\dinesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50661}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50661/devtoo..., se:cdpVersion: 124.0.6367.61, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (f556026a5996b2363f8140bb0913e400)] -&amp;gt; xpath: /html/body/div/div/div[1]/div[2]/div[2]/div/form/div[1]/label/span/div]
Session ID: f556026a5996b2363f8140bb0913e400">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: stale element reference: stale element not found in the current frame
  (Session info: chrome=124.0.6367.61)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Build info: version: '4.11.0', revision: '040bc5406b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '19.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [f556026a5996b2363f8140bb0913e400, clickElement {id=f.C0E64AD947F7FF20866C745F523CFD56.d.CD7FBF3D028BB7C7E9997D5F610F4B17.e.89}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 124.0.6367.61, chrome: {chromedriverVersion: 124.0.6367.60 (8771130bd84f..., userDataDir: C:\Users\dinesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50661}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50661/devtoo..., se:cdpVersion: 124.0.6367.61, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (f556026a5996b2363f8140bb0913e400)] -> xpath: /html/body/div/div/div[1]/div[2]/div[2]/div/form/div[1]/label/span/div]
Session ID: f556026a5996b2363f8140bb0913e400
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:224)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:77)
at Modules.BranchFields.uploadImage(BranchFields.java:49)
at stepDefinitions.BranchModule.enter_the_all_credentials_like1(BranchModule.java:144)
at ✽.Enter the all credentials like "afdd" "gds231" "fda@gamil.com" "89900" "78789" "12345" "iuhj" "jbnn" "jhjk" "bnkm" "76778" "76889"(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College1_Branch.feature:33)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="17.903">
    <failure type="java.lang.Exception" message="BugException">
      <![CDATA[java.lang.Exception: BugException
at stepDefinitions.BranchModule.it_need_to_show_error_message(BranchModule.java:195)
at ✽.It need to show error message.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College1_Branch.feature:45)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.499"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.26"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.966"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.077">
    <failure type="java.lang.AssertionError" message="expected [iuvvs] but found [ga]">
      <![CDATA[java.lang.AssertionError: expected [iuvvs] but found [ga]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.BranchModule1.check_the_values_present_in_all_fields(BranchModule1.java:53)
at ✽.Check the values present in all fields.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College2_Branch1.feature:18)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.728">
    <failure type="java.lang.AssertionError" message="expected [iuvvs] but found [ga]">
      <![CDATA[java.lang.AssertionError: expected [iuvvs] but found [ga]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.BranchModule1.check1(BranchModule1.java:66)
at ✽.Check the value present in all fields.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College2_Branch1.feature:23)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.306"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="19.98"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.211"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.845">
    <failure type="java.lang.Exception" message="DateMandatoryFieldDidn&amp;apos;tWork">
      <![CDATA[java.lang.Exception: DateMandatoryFieldDidn'tWork
at stepDefinitions.FacultyModule.check_how_it_behaves(FacultyModule.java:133)
at ✽.Check how it behaves.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College3_Faculty.feature:22)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.66"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="19.239">
    <failure type="java.lang.Exception" message="DateValidationMustInDOB">
      <![CDATA[java.lang.Exception: DateValidationMustInDOB
at stepDefinitions.FacultyModule.check_the_condition_is_true_or_false(FacultyModule.java:166)
at ✽.Check the condition is True or False.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College3_Faculty.feature:34)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.027">
    <failure type="java.lang.Exception" message="BugException">
      <![CDATA[java.lang.Exception: BugException
at stepDefinitions.FacultyModule.it_need_to_shows_error_message(FacultyModule.java:176)
at ✽.It need to shows error message.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College3_Faculty.feature:40)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.58"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.031"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.809"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.527"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.766"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.048">
    <failure type="java.lang.Exception" message="Error Message didn&amp;apos;t show">
      <![CDATA[java.lang.Exception: Error Message didn't show
at stepDefinitions.SubjectModule.error_message_showing_or_not_in_subject_page(SubjectModule.java:85)
at ✽.Error Message showing or not in subject page.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College4_Subject.feature:16)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.469">
    <failure type="java.lang.Exception" message="FacID is possible to able to clear">
      <![CDATA[java.lang.Exception: FacID is possible to able to clear
at stepDefinitions.SubjectModule.user_can_able_to_clear_fac_id_or_not(SubjectModule.java:94)
at ✽.User can able to clear facID or not.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College4_Subject.feature:21)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.726"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.133"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.388"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.386"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.348"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="18.262"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.644"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="20.442">
    <failure type="java.lang.Exception" message="To Validation Must">
      <![CDATA[java.lang.Exception: To Validation Must
at stepDefinitions.AcademicModule.if_the_data_is_present_throw_error(AcademicModule.java:111)
at ✽.If the data is present throw error.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College5_Academic.feature:21)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="22.982">
    <failure type="java.lang.Exception" message="Negative Value Accept in working days Fields">
      <![CDATA[java.lang.Exception: Negative Value Accept in working days Fields
at stepDefinitions.AcademicModule.if_it_allows_throw_error(AcademicModule.java:142)
at ✽.If it allows throw error.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College5_Academic.feature:27)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="21.936">
    <failure type="java.lang.Exception" message="validtion Must">
      <![CDATA[java.lang.Exception: validtion Must
at stepDefinitions.AcademicModule.enter_some_invalid_data_in_working_fields(AcademicModule.java:194)
at ✽.Enter some invalid data in working fields.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College5_Academic.feature:31)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.923"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.249">
    <failure type="java.lang.Exception" message="It created one data">
      <![CDATA[java.lang.Exception: It created one data
at stepDefinitions.AcademicModule.click_the_submit_and_add_button(AcademicModule.java:236)
at ✽.Click the submit and add button.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College5_Academic.feature:41)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.782">
    <failure type="java.lang.Exception" message="Values didn&amp;apos;t shows">
      <![CDATA[java.lang.Exception: Values didn't shows
at stepDefinitions.AcademicModule.check_all_the_values_present_or_not(AcademicModule.java:248)
at ✽.check all the values present or not.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College5_Academic.feature:45)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.696"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.726"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.441"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.671"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.881">
    <failure type="java.lang.Exception" message="Keyword Showing wrong">
      <![CDATA[java.lang.Exception: Keyword Showing wrong
at stepDefinitions.CourseModule.check_all_values_showing_or_not(CourseModule.java:124)
at ✽.Check all values showing or not.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College6_Course.feature:32)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.683"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.556"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.349"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.432">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="stale element reference: stale element not found in the current frame
  (Session info: chrome=124.0.6367.61)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Build info: version: &amp;apos;4.11.0&amp;apos;, revision: &amp;apos;040bc5406b&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;19.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [faf1093c42c805a6ddc8b591d42319fc, getElementText {id=f.BFC2DE6B5F9F94C0884FE19AC8F297C8.d.822C6044D0BB6264BF1DD4D6406BF827.e.88}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 124.0.6367.61, chrome: {chromedriverVersion: 124.0.6367.60 (8771130bd84f..., userDataDir: C:\Users\dinesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52415}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52415/devtoo..., se:cdpVersion: 124.0.6367.61, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (faf1093c42c805a6ddc8b591d42319fc)] -&amp;gt; xpath: //div[@id=&amp;apos;cell-3-undefined&amp;apos;]/div]
Session ID: faf1093c42c805a6ddc8b591d42319fc">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: stale element reference: stale element not found in the current frame
  (Session info: chrome=124.0.6367.61)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Build info: version: '4.11.0', revision: '040bc5406b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '19.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [faf1093c42c805a6ddc8b591d42319fc, getElementText {id=f.BFC2DE6B5F9F94C0884FE19AC8F297C8.d.822C6044D0BB6264BF1DD4D6406BF827.e.88}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 124.0.6367.61, chrome: {chromedriverVersion: 124.0.6367.60 (8771130bd84f..., userDataDir: C:\Users\dinesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52415}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52415/devtoo..., se:cdpVersion: 124.0.6367.61, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (faf1093c42c805a6ddc8b591d42319fc)] -> xpath: //div[@id='cell-3-undefined']/div]
Session ID: faf1093c42c805a6ddc8b591d42319fc
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:224)
at org.openqa.selenium.remote.RemoteWebElement.getText(RemoteWebElement.java:192)
at stepDefinitions.CourseModule.check_deleted_data_is_showing_or_not(CourseModule.java:202)
at ✽.Check deleted data is showing or not.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College6_Course.feature:50)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.464"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="28.375"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.068"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="29.423">
    <failure type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&amp;quot;method&amp;quot;:&amp;quot;xpath&amp;quot;,&amp;quot;selector&amp;quot;:&amp;quot;//div[text()=&amp;apos;Section ID is already exist&amp;apos;]&amp;quot;}
  (Session info: chrome=124.0.6367.61)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: &amp;apos;4.11.0&amp;apos;, revision: &amp;apos;040bc5406b&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;19.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [63363f3a5b2b4fcc7b389c73abfd579a, findElement {using=xpath, value=//div[text()=&amp;apos;Section ID is already exist&amp;apos;]}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 124.0.6367.61, chrome: {chromedriverVersion: 124.0.6367.60 (8771130bd84f..., userDataDir: C:\Users\dinesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52593}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52593/devtoo..., se:cdpVersion: 124.0.6367.61, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 63363f3a5b2b4fcc7b389c73abfd579a">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//div[text()='Section ID is already exist']"}
  (Session info: chrome=124.0.6367.61)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.11.0', revision: '040bc5406b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '19.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [63363f3a5b2b4fcc7b389c73abfd579a, findElement {using=xpath, value=//div[text()='Section ID is already exist']}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 124.0.6367.61, chrome: {chromedriverVersion: 124.0.6367.60 (8771130bd84f..., userDataDir: C:\Users\dinesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52593}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52593/devtoo..., se:cdpVersion: 124.0.6367.61, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 63363f3a5b2b4fcc7b389c73abfd579a
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:349)
at stepDefinitions.SectionModule.error_message_showing_or_not_in_section_menu(SectionModule.java:113)
at ✽.Error Message showing or not in section menu.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College7_Secyion.feature:28)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.117">
    <failure type="java.lang.Exception" message="User can able to clear ID data in Section">
      <![CDATA[java.lang.Exception: User can able to clear ID data in Section
at stepDefinitions.SectionModule.check_it_allow_or_not_in_section_menu(SectionModule.java:131)
at ✽.Check it allow or not in section menu.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College7_Secyion.feature:35)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.325">
    <failure type="java.lang.Exception" message="Negative Values allow in section ID fields">
      <![CDATA[java.lang.Exception: Negative Values allow in section ID fields
at stepDefinitions.SectionModule.check_the_if_the_data_is_present_throw_bugs(SectionModule.java:163)
at ✽.Check the if the data is present throw bugs.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College7_Secyion.feature:41)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.531"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.49"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.746"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.158"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.046">
    <failure type="java.lang.Exception" message="Deleted Not Successfuly">
      <![CDATA[java.lang.Exception: Deleted Not Successfuly
at stepDefinitions.SectionModule.check_deleted_data_is_showing_or_not_in_section_menu(SectionModule.java:250)
at ✽.Check deleted data is showing or not in section menu.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College7_Secyion.feature:63)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.005"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="31.876"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.13"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="31.82"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="29.628">
    <failure type="java.lang.Exception" message="Admission ID Validation Must">
      <![CDATA[java.lang.Exception: Admission ID Validation Must
at stepDefinitions.StudentModule.check_error_message_showing_or_not_like_admission_id_already_exists(StudentModule.java:147)
at ✽.Check error message showing or not like admission id already exists(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College8_Student.feature:29)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="24.38"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="20.712">
    <failure type="java.lang.Exception" message="Date Validation Didn&amp;apos;t Work">
      <![CDATA[java.lang.Exception: Date Validation Didn't Work
at stepDefinitions.StudentModule.check_the_values_present_in_student_page_or_not(StudentModule.java:232)
at ✽.Check the values present in student page or not.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College8_Student.feature:41)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="22.219"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.532">
    <failure type="java.lang.Exception" message="Validation Must in Email,Mobile No,Pincode">
      <![CDATA[java.lang.Exception: Validation Must in Email,Mobile No,Pincode
at stepDefinitions.StudentModule.enter_the_negative_values_in_email_mobile_no_pincode_etc(StudentModule.java:251)
at ✽.Enter the negative values in email,mobile no,pincode etc..(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College8_Student.feature:53)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.898"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="6.265"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.142"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.408"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.338"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="8.824">
    <failure type="org.openqa.selenium.ElementClickInterceptedException" message="element click intercepted: Element &amp;lt;button class=&amp;quot;MuiButtonBase-root Mui-disabled MuiPickersDay-root Mui-disabled MuiPickersDay-dayWithMargin css-6exafu&amp;quot; tabindex=&amp;quot;-1&amp;quot; type=&amp;quot;button&amp;quot; disabled=&amp;quot;&amp;quot; role=&amp;quot;gridcell&amp;quot; data-timestamp=&amp;quot;1713378600000&amp;quot; aria-colindex=&amp;quot;5&amp;quot; aria-selected=&amp;quot;false&amp;quot;&amp;gt;...&amp;lt;/button&amp;gt; is not clickable at point (1086, 525). Other element would receive the click: &amp;lt;div role=&amp;quot;row&amp;quot; class=&amp;quot;MuiDayCalendar-weekContainer css-mvmu1r&amp;quot; aria-rowindex=&amp;quot;3&amp;quot;&amp;gt;...&amp;lt;/div&amp;gt;
  (Session info: chrome=124.0.6367.61)
Build info: version: &amp;apos;4.11.0&amp;apos;, revision: &amp;apos;040bc5406b&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;19.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [34732c34b9bee2bb8a68eb225747be06, clickElement {id=f.547230019A8CF5306C4F16C37D064514.d.2B848704C85E8DBE2DBC42D1A4223B33.e.128}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 124.0.6367.61, chrome: {chromedriverVersion: 124.0.6367.60 (8771130bd84f..., userDataDir: C:\Users\dinesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:53588}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:53588/devtoo..., se:cdpVersion: 124.0.6367.61, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (34732c34b9bee2bb8a68eb225747be06)] -&amp;gt; xpath: /html/body/div[2]/div[2]/div/div/div/div[2]/div/div/div[2]/div/div[3]/button[5]]
Session ID: 34732c34b9bee2bb8a68eb225747be06">
      <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <button class="MuiButtonBase-root Mui-disabled MuiPickersDay-root Mui-disabled MuiPickersDay-dayWithMargin css-6exafu" tabindex="-1" type="button" disabled="" role="gridcell" data-timestamp="1713378600000" aria-colindex="5" aria-selected="false">...</button> is not clickable at point (1086, 525). Other element would receive the click: <div role="row" class="MuiDayCalendar-weekContainer css-mvmu1r" aria-rowindex="3">...</div>
  (Session info: chrome=124.0.6367.61)
Build info: version: '4.11.0', revision: '040bc5406b'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '19.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [34732c34b9bee2bb8a68eb225747be06, clickElement {id=f.547230019A8CF5306C4F16C37D064514.d.2B848704C85E8DBE2DBC42D1A4223B33.e.128}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 124.0.6367.61, chrome: {chromedriverVersion: 124.0.6367.60 (8771130bd84f..., userDataDir: C:\Users\dinesh\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:53588}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:53588/devtoo..., se:cdpVersion: 124.0.6367.61, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (34732c34b9bee2bb8a68eb225747be06)] -> xpath: /html/body/div[2]/div[2]/div/div/div/div[2]/div/div/div[2]/div/div[3]/button[5]]
Session ID: 34732c34b9bee2bb8a68eb225747be06
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:67)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:484)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:52)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:191)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:196)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:171)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:224)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:77)
at stepDefinitions.AnnouncementModule.user_enter_the_all_credentials(AnnouncementModule.java:53)
at ✽.User enter the all credentials(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College9_Announcement.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.048"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="9.808"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="5.36">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at stepDefinitions.HomePageDefinitions.user_can_able_to_edit_the_password_and_email_fields(HomePageDefinitions.java:115)
at ✽.User can able to edit the  password and email fields(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College_HomePage.feature:16)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.484">
    <failure type="java.lang.AssertionError" message="expected [Invalid Mobile Number] but found [Profile updated successfully]">
      <![CDATA[java.lang.AssertionError: expected [Invalid Mobile Number] but found [Profile updated successfully]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinitions.HomePageDefinitions.click_submit_icon_if_its_allow_there_is_a_bug(HomePageDefinitions.java:146)
at ✽.Click Submit icon  if its allow there is a bug.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College_HomePage.feature:22)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.516"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.81">
    <failure type="java.lang.Exception" message="All VAlues didn&amp;apos;t shows">
      <![CDATA[java.lang.Exception: All VAlues didn't shows
at stepDefinitions.HomePageDefinitions.check_all_values_showing_or_not_in_profile_view(HomePageDefinitions.java:173)
at ✽.Check all values showing or not in profile view.(file:///C:/Users/dinesh/eclipse-workspace/Colleges/src/test/java/features/College_HomePage.feature:31)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
